cli.py

```diff
@@ -590,10 +592,25 @@ def run_ratdecoders(file_path, passed_reporter):
         if key not in SUPER_LIST:
             others[key] = output[key]
     reporter.add_metadata("other", others)
-
     return {script_name: reporter.metadata}
 
 
+def run_mwcfg(file_path, reporter):
+    process = subprocess.run(['mwcfg', '--input', f'{file_path}', '-m', './modules'], capture_output=True)
+    output = ast.literal_eval(process.stdout.decode())
+    extracted = output[0]['configs'][0]
```

Comment:
since this is a hard list lookup followed by a hard dict lookup followed by a hard list lookup, are you sure that the same output format exists every time you run the `subprocess` command?

Reply:
as long as mwcfg is installed it should give the same output and I haven't seen any errors while testing so far, but I can wrap it in a try except anyway.

Reply:
Eh, no need. I guess we'll see if it ever errors by not wrapping it in a try-catch

Reply:
Maybe having a dict.get() on 'configs' might be a good idea since that's common error for some services unless you know it will always return that key

Reply:
I'm confident it always returns the configs dict. It should never change unless they change their code.